chains/substrate/executor/executor.go

```diff
@@ -47,6 +48,7 @@ type Executor struct {
 	fetcher     signing.SaveDataFetcher
 	bridge      BridgePallet
 	mh          MessageHandler
+	conn        *connection.Connection
```

Comment:
Conn will be extra here when we refactor it I assume, but lets leave it for now.